state-chain/pallets/cf-environment/src/lib.rs

```diff
@@ -453,6 +453,7 @@ impl<T: Config> Pallet<T> {
 						output_amount +
 							number_of_outputs * fee_per_output_utxo +
 							min_fee_required_per_tx,
+						Self::consolidation_parameters().consolidation_size,
```

Comment:
I wonder if this limit to too small. The number of utxos for consolidation is not as important as it is for egress. We were a little converative with consolidation since the only downside of setting a small limit is us paying more in fees. For egress, however, if we can't select utxos due to the limit, this would actually hurt user experience. We could probably be a little less conservative for egress and try, say 200, utxos at once and at least give a singing ceremony a chance to succeed (which it very well might).

There *is* a concern that we might be making ceremonies with 100+ payloads more likely (due to always selecting smallest utxos first), but in practice I doubt this will happen very often (a counter-argument to this could be that by allowing more payloads/utxos we won't need as many of them in the long term since we consolidate more efficiently).

Reply:
I was trying to avoid changing storage, but I think it is unavoidable.
I now renamed the consolidation_parameter to a more generic utxo_parameter, so the selection_limit can be configured separately to be more aggressive.